etc/qm/containers/containers.conf.d/qm_dropin_mount_bind_radio.conf

```diff
@@ -0,0 +1,29 @@
+# Drop-in configuration for Podman to mount bind /dev/radioX Stereo Radio
+#
+# In a typical vehicle system, radio is connected to car's onboard computer via a CAN bus
+# (Controller Area Network), which transmits signals from the radios to the car’s system for real-time
+# processing.
+#
+# However, it's possible to create a simulation environment using traditional hardware and open-source
+# software, eliminating the need for actual car radio or CAN bus integration. By using open-source
+# tools like Podman containers and radio processing libraries, virtual
+# radios can be simulated.
+#
+# "/dev/radio0:/dev/radio0", # Stereo Radio
+#
+#          Camera System Layout (Top-Down View)
+#
+#          ┌─────────────────────────────┐
+#          │        /dev/radio0          │
+#          │      (Stereo Radio)         │
+#          └────────────┬────────────────┘
+#                       │
+# ┌─────────────────────┴────────────────────────────────┐
+# │              Vehicle Body (Top View)                 │
+# │                                                      │
+# └──────────────────────────────────────────────────────┘
+#
+[containers]
```

Comment:
Please use quadlets, 
Us this as reference, the kvm use case for exposing device
https://github.com/containers/qm/blob/main/etc/containers/systemd/qm.container.d/qm_dropin_mount_bind_kvm.conf

Reply:
@Yarboa done